src/main/java/org/symphonyoss/integration/jira/webhook/parser/v2/JiraMetadataParser.java

```diff
@@ -523,6 +526,21 @@ private void processIconUrls(JsonNode input) {
   }
 
   /**
+   * Process custom field BaseUrl, uses the issue's link and removes the API call
+   * from the URL.
+   * @param input JSON node that contains user information provided by JIRA
+   */
+  private void processBaseUrl(JsonNode input) {
+    ObjectNode issueNode = (ObjectNode) input.path(ISSUE_PATH);
```

Comment:
You don't need to cast the issueNode variable to ObjectNode.

I think you could change to:
```
String selfPath = input.path(ISSUE_PATH).path(SELF_PATH).asText();
selfPath = selfPath.replaceAll(BASE_API_REGEX,StringUtils.EMPTY);

((ObjectNode) input).(BASE_URL, selfPath)
```